renderer/pages/editor.js

```diff
@@ -1,10 +1,107 @@
+import path from 'path';
 import React from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+import TrafficLights from '../components/traffic-lights';
+import Options from '../components/editor/options';
+import Video from '../components/editor/video';
+
+class Editor extends React.Component {
+  static propTypes = {
+    src: PropTypes.string.isRequired
+  }
 
-export default class extends React.Component {
   render() {
+    const {src} = this.props;
     return (
       <div>
-        <span>This is the editor</span>
+        <div className="video-container">
+          <div className="title-bar">
+            <TrafficLights/>
+            <span className="title-bar__title">kap-beta.mp4</span>
+          </div>
+          <Video
+            src={src}
```

Comment:
no need to split into multiple lines

Reply:
That's how xo wants it I think, I'll try to put it on one line.